lib/generators/react_on_rails/base_generator.rb

```diff
@@ -73,7 +73,6 @@ def copy_webpacker_config
       end
 
       def add_base_gems_to_gemfile
-        gem "mini_racer", platforms: :ruby
```

Comment:
We should have a changelog entry reflecting that. If users already have `mini_racer` in their Gemfiles, nothing will change. We should note that it's been removed and ask users to also remove and test their apps for any changes